package.json

```diff
@@ -1620,7 +1621,7 @@
     "postcss-loader": "^4.2.0",
     "postcss-prefix-selector": "^1.16.0",
     "postcss-scss": "^4.0.4",
-    "prettier": "^2.7.1",
+    "prettier": "2.7.1",
```

Comment:
TLDR; The updates triggered a minor bump to prettier which in turn caused lint changes that aren't related to the PR to be included by the CI. Please the PR description for more info, and the plan to revert this.

Reply:
It sounds like a very peculiar way to solve this. Can't you just manually revert Prettier-related changes in yarn.lock and then re-run `yarn kbn bootstrap` to reconcile the lock file (you can do this manually too)? As a result, your yarn.lock should look like this:

```diff
diff --git a/yarn.lock b/yarn.lock

-prettier@2.0.5:
-  version "2.0.5"
-  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.0.5.tgz#d6d56282455243f2f92cc1716692c08aa31522d4"
-  integrity sha512-7PtVymN48hGcO4fGjybyBSIWDsLU4H4XlvOHfq91pz9kkGlonzwTfYkaIEwiRg/dAJF9YlbsduBAgtYLi+8cFg==
-
 "prettier@>=2.2.1 <=2.3.0":
-  version "2.2.1"
-  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.2.1.tgz#795a1a78dd52f073da0cd42b21f9c91381923ff5"
-  integrity sha512-PqyhM2yCjg/oKkFPtTGUojv7gnZAoG80ttl45O6x2Ug/rMJw4wcc9k6aaf2hibP7BGVCCM33gZoGjyvt9mm16Q==
+  version "2.3.0"
+  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.3.0.tgz#b6a5bf1284026ae640f17f7ff5658a7567fc0d18"
+  integrity sha512-kXtO4s0Lz/DW/IJ9QdWhAf7/NmPWQXkFr/r/WkR3vyI+0v8amTDxiaQSLzs8NBlytfLWX/7uQUMIW677yLKl4w==
 
-prettier@^2.7.1:
+prettier@^2.0.0, prettier@^2.7.1:
   version "2.7.1"
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.7.1.tgz#e235806850d057f97bb08368a4f7d899f7760c64"
   integrity sha512-ujppO+MkdPqoVINuDFDRLClm7D78qbDt0/NR+wp5FqEZOoTNAjPHWj17QRhu7geIHJfcNhRk1XVQmF8Bp3ye+g==
```

Reply:
Hey @azasypkin thanks for the pointer, I might be missing something but this doesn't work primarily because monaco-yaml has a dependency for prettier version 2.8.8, so reverting the lock file to match the definition for prettier on main without locking down the version doesn't work because an install would just add changes to the lock file, and deciding to push as is would also result in an error in during the build in the CI as far as I know.

Reply:
> monaco-yaml has a dependency for prettier version 2.8.8

Hmm, I see that `monaco-yaml` requires `prettier "^2.0.0"` (so it should work well with `2.7.1`), where do you see that it needs `2.8.8`?

> an install would just add changes to the lock file

What `install` do you mean exactly? `yarn kbn install` won't alter `yarn.lock` without a strong reason (unless you use `yarn add` or something along these lines that you shouldn't as it usually does what we don't ask it). Have you tried making the changes I suggested above manually and re-running `yarn kbn bootstrap`?

Reply:
> Have you tried making the changes I suggested above manually and re-running yarn kbn bootstrap?

Yes I did, I should have been clearer about that. The issue is after applying the generated patch which quite alright has us at version 2.7.1, running `yarn kbn bootstrap` triggers a modification to the lockfile  because `monaco-yaml` resolves it's dependency of prettier to 2.8.8 which is appropriate seeing that the project will accept the latest minor version. 

Running `yarn why prettier` yields the following;

<img width="641" alt="Screenshot 2024-01-22 at 10 59 44" src="https://github.com/elastic/kibana/assets/7893459/39a525e4-dda7-4f84-a4d6-f6216891109f">

Reply:
I see, it's not how I expected this to work. Would you mind if I push the suggestion to your PR branch?

Reply:
Oh, wait, your current `yarn.lock` already looks correct, isn't CI happy or ...?

Reply:
Please, be my guest